admin/schroot/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=schroot
+PKG_REV:=2ba1e043dbd65c8f08d1d4821393b099af05af15
+PKG_VERSION:=1.6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://salsa.debian.org/debian/schroot
+PKG_MIRROR_HASH:=4df3adb05e24611f124376f2dd708413751535caf1d1d1b1acbd91d712ea843d
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=$(PKG_REV)
+
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf gettext-version
+PKG_BUILD_DEPENDS:=gettext
+
+include $(INCLUDE_DIR)/autotools.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/schroot
+  SECTION:=admin
+  CATEGORY:=Administration
+  TITLE:=Securely enter a chroot and run a command or login shell.
+  DEPENDS:=+boost +boost-filesystem +boost-iostreams +boost-program_options +boost-regex +libstdcpp \
+	+SCHROOT_BTRFS:btrfs-progs +SCHROOT_LOOPBACK:losetup +SCHROOT_LVM:lvm2 +SCHROOT_UUID:libuuid
+  URL:=https://salsa.debian.org/debian/schroot
+endef
+
+define Package/schroot/description
+Securely enter a chroot and run a command or login shell.
+endef
+
+define Package/sudo/conffiles
+/etc/schroot/
+endef
+
+define Package/schroot/config
+	source "$(SOURCE)/Config.in"
+endef
+
+PKG_CONFIG_DEPENDS := \
+	CONFIG_SCHROOT_BTRFS \
+	CONFIG_SCHROOT_LOOPBACK \
+	CONFIG_SCHROOT_LVM \
+	CONFIG_SCHROOT_UUID
+
+CONFIGURE_ARGS += \
+	--enable-block-device \
+	--enable-union \
+	--disable-dchroot \
+	--disable-dchroot-dsa \
+	--disable-csbuild \
+	--disable-debug \
+	--disable-pam \
+	--disable-doxygen \
+	--disable-rpath
+
+ifeq ($(CONFIG_SCHROOT_BTRFS),)
```

Comment:
You could consolidate all of these with:

$(if $(CONFIG_SCHROOT_BTRFS),--enable,--disable)-btrfs-snapshot

added to configure_args.